js/ajaxTask.js

```diff
@@ -0,0 +1,34 @@
+function send() {
+    let username = getElementByIdFromDocument("username").value;
+    let password = getElementByIdFromDocument("password").value;
+    let confirmPassword = getElementByIdFromDocument("confirmPassword").value;
+
+    let object = { username, password, confirmPassword };
+    let jsonToBeSent = converObjectToJSON(object);
+
+    const url = "./register.php"
+    let callback = function (text) {
+        console.log(text);
+    };
+
+    ajax(url, { success: callback, data: jsonToBeSent });
+}
+function getElementByIdFromDocument(elementName) {
```

Comment:
I would say that you could remove the following 2 functions if you want. They do not really add a lot. It also makes it a bit harder to read the code as you need to check the functions in order to see what they are doing.